dags/atd_service_bot_index_issues_to_dts_portal.py

```diff
@@ -34,6 +34,7 @@
     "GITHUB_ACCESS_TOKEN": {
         "opitem": "Github Access Token Service Bot",
         "opfield": ".password",
+
```

Comment:
This change snuck in here in here 👀

Reply:
Wow, nice catch! I looked, and I brought that in when I merged `airflow-v2` into this branch at some point, and then it was corrected in `airflow-v2` since. Great catch; this would have undone someone's work!